src/components/MapContainer/MapContainer.tsx

```diff
@@ -3,16 +3,31 @@ import LayersDrawer from '../LayersDrawer/LayersDrawer'
 import BaseMap from '../BaseMap/BaseMap'
 import RegionSelect from '../RegionSelect/RegionSelect'
 import styles from './MapContainer.module.scss'
+import YearSelect from '../YearSelect/YearSelect'
+import useResponsive from '../../hooks/useResponsive'
 
 export default function MapContainer() {
+  const { isMobileWidth } = useResponsive()
   const [layerOn, setLayerOn] = useState(true)
+  const [selectedYear, setSelectedYear] = useState(2020)
 
   return (
     <div className={styles['MapContainer-root']}>
-      <div className={styles['layer-controls']}>
-        <LayersDrawer layerOn={layerOn} setLayerOn={setLayerOn} />
-        <RegionSelect />
-      </div>
+      {isMobileWidth ? (
+        <div className={styles['layer-controls-mobile']}>
+          <LayersDrawer layerOn={layerOn} setLayerOn={setLayerOn} />
+          <RegionSelect />
+          <YearSelect selectedYear={selectedYear} onChange={setSelectedYear} />
+        </div>
+      ) : (
+        <div className={styles['layer-controls-desktop']}>
```

Comment:
If the only difference between the renderings is the style class, a conditional statement can be used to reduce duplicate code
```suggestion
        <div className={styles[`layer-controls--${isMobileWidth ? 'mobile' : 'desktop'}`]}>
```